src/main/java/kr/polymarket/global/util/S3Uploader.java

```diff
@@ -31,18 +26,21 @@ public class S3Uploader {
     @Value("${cloud.aws.s3.bucket}")
     private String bucket;
 
+    @Value("${cloud.aws.s3.url}")
+    private String awsUrl;
+
     /**
      * 정책 고려사항
      * 1. 파일 확장자, 실제 파일타입 일치 여부
      * 2. 파일 사이즈 제한 얼마나 할 것인가? 단위 : 픽셀(당근마켓은 제한 없음)
      */
 
-    public List<String> uploadImage(List<MultipartFile> multipartFile)  {
+    public List<String> uploadProductImage(List<MultipartFile> multipartFile) {
```

Comment:
S3Uploader는 유틸성 함수인데 uploadProductImage라는 이름은 비즈니스 로직의 의미가 담긴거라 작명이 적절치 않아보여
그리고 uploadImage라는 이름으로 오버로딩 하는게 더 객체지향스럽다고 생각해
```suggestion
    public List<String> uploadProduct(List<MultipartFile> multipartFile) {
```